tests/test_feedexport.py

```diff
@@ -1469,6 +1472,496 @@ def test_extend_kwargs(self):
             self.assertEqual(row['expected'], data[feed_options['format']])
 
 
+class FeedPostProcessedExportsTest(FeedExportTestBase):
+    __test__ = True
+
+    items = [{'foo': 'bar'}]
+    expected = b'foo\r\nbar\r\n'
+
+    class MyPlugin1:
+        def __init__(self, file, feed_options):
+            self.file = file
+            self.feed_options = feed_options
+            self.char = self.feed_options.get('plugin1_char', b'')
+
+        def write(self, data):
+            return self.file.write(bytes(data) + self.char)
```

Comment:
Pypy's pickle implementation was writing data in `memoryview` instead of expected `bytes`. So I was getting operand error. I figured converting it to `bytes`.

Reply:
Then what about moving the casting to the Pickle plugin and adding a comment about this? (hopefully there’s an online bug report or note somewhere that we can link)

Reply:
The `write` function of the custom plugin is being used by the `pickle.dump` method in `PickleItemExporter`. Can't really move the casting there in the `dump` method. And I can't find anything about the `memoryview` problem either. Seems like a very rare case someone would use a wrapper in the `dump` method :sweat_smile:.

Reply:
Does it work if you use 2 separate write calls?

```python
written_count = self.file.write(data)
written_count += self.file.write(self.char)
return written_count
```

Reply:
> Does it work if you use 2 separate write calls?

Yup it works that way. It also works like this: `self.file.write(self.char + data)` weirdly.

> If we go this route, it might be good to indicate that the input data may be a memoryview, and support that officially.

By that you mean we should include in the documentation to warn the users that input could be a `memoryview` while using `PickleItemExporter`?

Reply:
I would not put the focus on `PickleItemExporter`, and instead assume that any exporter, built-in or user-provided, may do the same.

Since supporting a `memoryview` should be trivial, and we are going to support it in our post-processing plugins, it would make sense to indicate in the documentation of plugins that their `write` method may get either `memoryview` or `bytes`, and that they should be ready to handle any of those types as input.